### PR TITLE
Improve performance of transfer_attribute_drillpaths()

### PR DESCRIPTION
**Pull request by korczis**
## Todos
- [ ] Tests
- [ ] Documentation
- [ ] Changelog
- [ ] Build passing

### CODE REVIEW

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-rubocop-zuul/635/ : SUCCESS in 11s
- https://ci.intgdc.com/job/gooddata-ruby-tests-zuul/586/ : ABORTED in 2h 00m 01s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/355/ : UNSTABLE in 5m 56s

**Comment by panjan** [comment]
ok to test

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-rubocop-zuul/642/ : SUCCESS in 9s
- https://ci.intgdc.com/job/gooddata-ruby-tests-zuul/593/ : FAILURE in 12m 54s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/362/ : UNSTABLE in 7m 42s

File: lib/gooddata/lcm/actions/synchronize_attribute_drillpaths.rb [hunk 1]
```diff
@@ -45,7 +45,8 @@ def call(params)
               client.projects(pid) || fail("Invalid 'to' project specified - '#{pid}'")
             end
 
-            GoodData::LCM.transfer_attribute_drillpaths(from_project, to_projects)
+            attributes = from_project.attributes
+            GoodData::LCM.transfer_attribute_drillpaths(from_project, to_projects, attributes)
```
Comment (vhtien) [inline lib/gooddata/lcm/actions/synchronize_attribute_drillpaths.rb]: Could you tell me why we can enhance the performance by this solution? AFAIK, we only have ONE source project and MANY target projects, so caching the source attributes here is the same as what we've done in `GoodData::LCM.transfer_attribute_drillpaths`, that means we enhance nothing.

**Pull request closed without merging**